static/js/place/chart_block.tsx

```diff
@@ -96,96 +96,67 @@ class ChartBlock extends React.Component<ChartBlockPropType, unknown> {
       );
     }
 
-    // Only add comparison chart for US places.
-    if (this.props.isUsaPlace) {
-      // Prepare parameters for related charts.
-      let unit = this.props.data.unit;
-      let scaling = this.props.data.scaling;
-      const relatedChart = this.props.data.relatedChart;
-      if (relatedChart && relatedChart.scale) {
-        unit = relatedChart.unit;
-        scaling = relatedChart.scaling ? relatedChart.scaling : 1;
-      }
-      const chartType =
-        this.props.data.statsVars.length == 1
-          ? chartTypeEnum.STACK_BAR
-          : chartTypeEnum.GROUP_BAR;
-      const displayPlaceType = displayNameForPlaceType(
-        this.props.placeType,
-        true /* isPlural */
-      ).toLocaleLowerCase();
+    // Prepare parameters for related charts.
```

Comment:
github is giving bad diff. This is just to remove the "if (this.props.isUsaPlace)" check